FrontEnd/src/components/authorization/LoginContent.jsx

```diff
@@ -1,10 +1,25 @@
 import { useForm } from "react-hook-form";
 import { useState, useEffect} from "react";
+import axios from 'axios';
 import validator from "validator";
 import EyeVisible from "./EyeVisible";
 import EyeInvisible from "./EyeInvisible";
 import classes from "./LoginContent.module.css";
 
+axios
```

Comment:
I think react-query could be the easiest option for us. You may follow [this tutorial ](https://dev.to/this-is-learning/react-query-authentication-flow-id2)